### PR TITLE
Libdai

### PR DESCRIPTION
**Pull request by DerThorsten**

### CODE REVIEW

File: recipes/libdai/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - boost-cpp >=1.4
```
Comment (jakirkham) [inline recipes/libdai/recipe/meta.yaml]: Are we only using the headers or is this using Boost libraries too?

File: recipes/libdai/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/dai  # [unix]
+    - test -f ${PREFIX}/include/dai/alldai.h  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\dai\alldai.h (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/DerThorsten/libdai
+  license: BSD
```
Comment (jakirkham) [inline recipes/libdai/recipe/meta.yaml]: Please add version of BSD. Appears like BSD 2-Clause, but could be wrong.

File: recipes/libdai/recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = "0.3.3" %}
+{% set sha256 = "9ba07a11af50bb4d53bceade6295a0366ef90e92440690ac52935ec144c05c70" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: libdai-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/libdai/archive/{{ version }}.tar.gz
```
Comment (jakirkham) [inline recipes/libdai/recipe/meta.yaml]: So it appears this a fork and the original source is [here]( https://bitbucket.org/jorism/libdai ). Though it appears to have been unmaintained for 1.5yrs now. The last release being nearly 3yrs ago. Seems reasonable to fork and update IMHO. Thoughts from others?

cc @conda-forge/core @conda-forge/staged-recipes

File: recipes/libdai/recipe/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - boost-cpp >=1.4
+    - gmp
+  run:
+    - gmp
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/dai  # [unix]
+    - test -f ${PREFIX}/include/dai/alldai.h  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\dai\alldai.h (exit 0) else (exit 1)  # [win]
```
Comment (jakirkham) [inline recipes/libdai/recipe/meta.yaml]: Any libraries to check for here?

File: recipes/libdai/recipe/build.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake  -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+make install
```
Comment (jakirkham) [inline recipes/libdai/recipe/build.sh]: Is there a `make check` that we can run?

File: recipes/libdai/recipe/bld.bat [hunk 6]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles"  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
```
Comment (jakirkham) [inline recipes/libdai/recipe/bld.bat]: Is there a `nmake check` that we can run?

File: recipes/libdai/recipe/meta.yaml (hunk 3 above)
Comment (CJ-Wright) [reply recipes/libdai/recipe/meta.yaml]: That seems reasonable.

File: recipes/libdai/recipe/bld.bat (hunk 6 above)
Comment (DerThorsten) [reply recipes/libdai/recipe/bld.bat]: I can add a  `make check` if appreciated once I am back from vacations

**Pull request closed without merging**